metricbeat/module/elasticsearch/ml_job/_meta/data.json

```diff
@@ -1,33 +1,105 @@
 {
     "@timestamp": "2017-10-12T08:05:34.853Z",
-    "agent": {
-        "hostname": "host.example.com",
-        "name": "host.example.com"
-    },
     "elasticsearch": {
         "cluster": {
-            "id": "3LbUkLkURz--FR-YO0wLNA",
-            "name": "es1"
+            "id": "8l_zoGznQRmtoX9iSC-goA",
+            "name": "docker-cluster"
         },
         "ml": {
             "job": {
                 "data_counts": {
-                    "invalid_date_count": 0,
-                    "processed_record_count": 0
```

Comment:
These two changes look breaking to me. Looks like `invalid_date_count` was replaced by `invalid_date.count` and `processed_record_count` was replaced by `record.processed.count`. How about we add field aliases for these so the changes are non-breaking, at least for users relying on these fields for filtering and aggregations? Or if you want to be completely safe, you can leave these two fields as-is and we can remove them later in an 8.0-only PR (one that won't be backported to 7.x).

Reply:
Thank you for spotting this. I think I changed it in the first commit before you warned me the first time. I have reverted it and we will see what we can / should change on 8.0 (if any)